config.md

```diff
@@ -302,13 +307,13 @@ For Windows based systems the user structure has the following fields:
 
 **`platform`** (object, REQUIRED) specifies the configuration's target platform.
 
-* **`os`** (string, REQUIRED) specifies the operating system family of the container configuration's specified [`root`](#root) file system bundle.
+* **`os`** (string, REQUIRED) specifies the operating system family of the container configuration's specified [`root`](#root) filesystem bundle.
   The runtime MUST generate an error if it does not support the specified **`os`**.
-  Bundles SHOULD use, and runtimes SHOULD understand, **`os`** entries listed in the Go Language document for [`GOOS`][go-environment].
+  Value for **`os`** SHOULD use, and runtimes SHOULD understand, **`os`** entries listed in the Go Language document for [`GOOS`][go-environment].
```

Comment:
I don't think readers are likely to forget which property we're talking about ;).  Can we shorten this line?  Maybe:

    Values should be, and runtimes SHOULD understand, valid [`GOOS`][go-environment] values.